.eslintrc.json

```diff
@@ -34,7 +34,8 @@
     "import/named": 2,
     "import/no-named-as-default": 0,
     "import/no-named-as-default-member": 0,
-    "import/order": 2
+    "import/order": 2,
+    "react/prop-types": "off"
```

Comment:
Seems like we will get an error with `prop-types` when we use `React.forwardRef` to render our component.

<img width="616" alt="Screen Shot 2022-03-20 at 11 58 17" src="https://user-images.githubusercontent.com/25715018/159148862-a7185d74-754c-4140-94c2-1a3ca5814f9c.png">

I'm disabling the rule for the entire codebase as I found that we have already disabled this rule in a couple places in /client. And since we have been moving to TypeScript, I don't think we need to care about React `prop-types` anymore.

Reply:
I do not mind disabling this, because we only have 4 files left that use the `prop-types` library, and 2 of those cannot be migrated (yet).